### ISSUE TITLE
Documentation: connection to Lustre

### ISSUE THREAD

**Issue by ulysses4ever** [OctopiChalmers/haski#1]
Hi! I watched your talk at Haskell Symposium 2020 -- good job! You mentioned Lustre several times. Especially it seemed crucial for the answer to the question about handling temperature hysteresis ("if you can solve it in Lustre, you can solve it in Haski"). Is it the file system? Would it make sense to mention it in README? If not, feel free to close the issue.

**Comment by Abhiroop** [comment]
@ulysses4ever I believe the authors were referring to Lustre the programming language - http://www-verimag.imag.fr/The-Lustre-Programming-Language-and and not the file system.

**Comment by ulysses4ever** [comment]
@Abhiroop thanks for the clarification! I hope this could be added to the documentation.

**Comment by Abhiroop** [comment]
Yes I agree, this is important for the documentation.

**Comment by nachivpn** [comment]
That's right, the reference is to the programming language Lustre. Interestingly, this comment was also given on the paper by one of the reviewers! I'll make a note of this in the README to avoid this confusion.
